{"uid":"a00e01232b23987c","name":"测试数据：【 wintest，123456，True，0，登录成功】","fullName":"testcases.api_test.test_03_login.TestUserLogin#test_login_user","historyId":"4ab02ebad4baabe28d2fb7352ad2343a","time":{"start":1744699122724,"stop":1744699122820,"duration":96},"description":"该用例是针对获取用户登录接口的测试","descriptionHtml":"<p>该用例是针对获取用户登录接口的测试</p>\n","status":"failed","statusMessage":"AssertionError: 接口返回码是 【 1003 】, 返回信息：用户名不存在！！！ \nassert False == True\n +  where False = <core.result_base.ResultBase object at 0x10a16fb50>.success","statusTrace":"self = <testcases.api_test.test_03_login.TestUserLogin object at 0x10a1c1970>\nusername = 'wintest', password = '123456', except_result = True, except_code = 0\nexcept_msg = '登录成功'\n\n    @allure.story(\"用例--登录用户\")\n    @allure.description(\"该用例是针对获取用户登录接口的测试\")\n    @allure.issue(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应BUG的链接地址\")\n    @allure.testcase(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应用例的链接地址\")\n    @allure.title(\"测试数据：【 {username}，{password}，{except_result}，{except_code}，{except_msg}】\")\n    @pytest.mark.single\n    @pytest.mark.parametrize(\"username, password, except_result, except_code, except_msg\",\n                             api_data[\"test_login_user\"])\n    def test_login_user(self, username, password, except_result, except_code, except_msg):\n        logger.info(\"*************** 开始执行用例 ***************\")\n        result = login_user(username, password)\n        step_1(username)\n>       assert result.success == except_result, result.error\nE       AssertionError: 接口返回码是 【 1003 】, 返回信息：用户名不存在！！！ \nE       assert False == True\nE        +  where False = <core.result_base.ResultBase object at 0x10a16fb50>.success\n\ntestcases/api_test/test_03_login.py:30: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"description":"该用例是针对获取用户登录接口的测试","status":"failed","statusMessage":"AssertionError: 接口返回码是 【 1003 】, 返回信息：用户名不存在！！！ \nassert False == True\n +  where False = <core.result_base.ResultBase object at 0x10a16fb50>.success","statusTrace":"self = <testcases.api_test.test_03_login.TestUserLogin object at 0x10a1c1970>\nusername = 'wintest', password = '123456', except_result = True, except_code = 0\nexcept_msg = '登录成功'\n\n    @allure.story(\"用例--登录用户\")\n    @allure.description(\"该用例是针对获取用户登录接口的测试\")\n    @allure.issue(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应BUG的链接地址\")\n    @allure.testcase(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应用例的链接地址\")\n    @allure.title(\"测试数据：【 {username}，{password}，{except_result}，{except_code}，{except_msg}】\")\n    @pytest.mark.single\n    @pytest.mark.parametrize(\"username, password, except_result, except_code, except_msg\",\n                             api_data[\"test_login_user\"])\n    def test_login_user(self, username, password, except_result, except_code, except_msg):\n        logger.info(\"*************** 开始执行用例 ***************\")\n        result = login_user(username, password)\n        step_1(username)\n>       assert result.success == except_result, result.error\nE       AssertionError: 接口返回码是 【 1003 】, 返回信息：用户名不存在！！！ \nE       assert False == True\nE        +  where False = <core.result_base.ResultBase object at 0x10a16fb50>.success\n\ntestcases/api_test/test_03_login.py:30: AssertionError","steps":[{"name":"步骤1 ==>> 登录用户","time":{"start":1744699122820,"stop":1744699122820,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"username","value":"'wintest'"}],"stepsCount":0,"hasContent":true,"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false}],"attachments":[{"uid":"c0fd43edc0900be7","name":"log","source":"c0fd43edc0900be7.txt","type":"text/plain","size":953},{"uid":"bf91333d2a71b46d","name":"stderr","source":"bf91333d2a71b46d.txt","type":"text/plain","size":1185}],"parameters":[],"stepsCount":1,"hasContent":true,"shouldDisplayMessage":true,"attachmentsCount":2,"attachmentStep":false},"afterStages":[],"labels":[{"name":"severity","value":"normal"},{"name":"epic","value":"针对单个接口的测试"},{"name":"feature","value":"用户登录模块"},{"name":"story","value":"用例--登录用户"},{"name":"tag","value":"single"},{"name":"parentSuite","value":"testcases.api_test"},{"name":"suite","value":"test_03_login"},{"name":"subSuite","value":"TestUserLogin"},{"name":"host","value":"192.168.10.5"},{"name":"thread","value":"23339-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"testcases.api_test.test_03_login"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"except_code","value":"0"},{"name":"except_msg","value":"'登录成功'"},{"name":"except_result","value":"True"},{"name":"password","value":"'123456'"},{"name":"username","value":"'wintest'"}],"links":[{"name":"点击，跳转到对应用例的链接地址","url":"https://www.cnblogs.com/wintest","type":"test_case"},{"name":"点击，跳转到对应BUG的链接地址","url":"https://www.cnblogs.com/wintest","type":"issue"}],"hidden":true,"retry":true,"extra":{"categories":[],"tags":["single"]},"source":"a00e01232b23987c.json","parameterValues":["0","'登录成功'","True","'123456'","'wintest'"]}